configure

```diff
@@ -686,6 +686,7 @@ infodir
 docdir
```

Comment:
auto-generated changes - ignore in your review
see below in file developer_notes for more infos